content_type_middleware.go

```diff
@@ -0,0 +1,12 @@
+package handlers
+
+import (
+	"net/http"
+)
+
+var JSONContentTypeMiddleware = MiddlewareFunc(func(handler HandlerFunc) HandlerFunc {
```

Comment:
suggestion: IMO this would be a better name in case we ever add another middleware for another `Content-Type`.
```suggestion
var ContentTypeJSONMiddleware = MiddlewareFunc(func(handler HandlerFunc) HandlerFunc {
```